src/app/Component/chatroom/chatroom.css

```diff
@@ -1,137 +1,137 @@
+  margin-left: 5px !important;
 }
 .msg-time-right{
-    margin-right: 15px;
-    color: #828282;
-    font-style: italic;
-    font-size: 12px;
+  margin-right: 15px;
+  color: #828282;
+  font-style: italic;
+  font-size: 12px;
 }
 .msg-time-left{
-    margin-left: 5px;
-    color: #828282;
-    font-style: italic;
-    font-size: 12px;
-}
-
-  .bounce {
-  position: absolute;
-  bottom: 8%;
-  left: 16px ;
-  width: 5px;
-  height: 5px ;
-    background: #fff;
-  -webkit-border-radius:50%;
-  -moz-border-radius:50%;
-  -ms-border-radius:50%;
-  border-radius:50%;
-  animation: bounce 1s infinite;
-  -webkit-animation: bounce 1s infinite;
-  -moz-animation: bounce 1s infinite;
-  -o-animation: bounce 1s infinite;
-  }
-  
-  .bounce1 {
-  position: absolute;
-  bottom: 8%;
-  left: 29px ;
-  width: 5px;
-  height: 5px ;
-    background: #fff;
-  -webkit-border-radius:50%;
-  -moz-border-radius:50%;
-  -ms-border-radius:50%;
-  border-radius:50%;
-  animation: bounce 1s infinite;
-  -webkit-animation: bounce 1s infinite;
-  -moz-animation: bounce 1s infinite;
-  -o-animation: bounce 1s infinite;
-    animation-delay: 0.2s;
-  }
-  
-  .bounce2 {
-  position: absolute;
-  bottom: 8%;
-  left: 5%;
-  width: 5px;
-  height: 5px ;
-    background: #fff;
-  -webkit-border-radius:50%;
-  -moz-border-radius:50%;
-  -ms-border-radius:50%;
-  border-radius:50%;
-  animation: bounce 1s infinite;
-  -webkit-animation: bounce 1s infinite;
-  -moz-animation: bounce 1s infinite;
-  -o-animation: bounce 1s infinite;
-    animation-delay: 0.4s;
-  }
-   
-  @-webkit-keyframes bounce {
-  0%, 20%, 50%, 80%, 100% {-webkit-transform: translateY(0);}
-  40% {-webkit-transform: translateY(-10px);}
-  }
-   
-  @-moz-keyframes bounce {
-  0%, 20%, 50%, 80%, 100% {-moz-transform: translateY(0);}
-  40% {-moz-transform: translateY(-10px);}
-  }
-   
-  @-o-keyframes bounce {
-  0%, 20%, 50%, 80%, 100% {-o-transform: translateY(0);}
-  40% {-o-transform: translateY(-10px);}
-  }
-  @keyframes bounce {
-  0%, 20%, 50%, 80%, 100% {transform: translateY(0);}
-  40% {transform: translateY(-10px);}
-  
-  }
-
-  .scroll{
-      overflow-y: auto;
-      margin-right: -20px;
-  }
-  .chatroom-date{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
-
-  .messagepopup-right{
-    height: 12rem;
-    width: 73%;
-    margin-right: 5rem; 
-     margin-top: 8px;
-    background: #50535a;
-    box-shadow: 0px 0px 30px rgb(14 10 16 / 84%);
-    border-radius: 10px;
+  margin-left: 5px;
+  color: #828282;
+  font-style: italic;
+  font-size: 12px;
 }
 
-.messagepopup-left{
+.bounce {
+position: absolute;
+bottom: 8%;
+left: 16px ;
+width: 5px;
+height: 5px ;
+  background: #fff;
+-webkit-border-radius:50%;
+-moz-border-radius:50%;
+-ms-border-radius:50%;
+border-radius:50%;
+animation: bounce 1s infinite;
+-webkit-animation: bounce 1s infinite;
+-moz-animation: bounce 1s infinite;
+-o-animation: bounce 1s infinite;
+}
+
+.bounce1 {
```

Comment:
what does this **bounce1** name refer for?

Reply:
when opponent is typing user will get 3 dots bouncing,bounce1 is for 3rd dot